docs/content/docs/dev/configuration/connector.md

```diff
@@ -24,39 +24,42 @@ under the License.
 
 # Connectors and Formats
 
-Flink can read from and write to various external systems via connectors and define the format in 
-which to store the data.
+Flink can read from and write to various external systems via connectors and use the format of your choice
+in order to read/write data from/into records.
 
-The way that information is serialized is represented in the external system and that system needs
-to know how to read this data in a format that can be read by Flink.  This is done through format 
-dependencies.
+An overview of available connectors and formats is available for both
+[DataStream]({{< ref "docs/connectors/datastream/overview.md" >}}) and
+[Table API/SQL]({{< ref "docs/connectors/table/overview.md" >}}).
 
-Most applications need specific connectors to run. Flink provides a set of formats that can be used 
-with connectors (with the dependencies for both being fairly unified). These are not part of Flink's 
-core dependencies and must be added as dependencies to the application.
+In order to use connectors and formats, you need to make sure Flink has access to the artifacts implementing them. 
+For each connector supported by the Flink community, we publish on [Maven Central](https://search.maven.org) two artifacts:
 
-## Adding Dependencies 
+* `flink-connector-<NAME>` which is a thin JAR including only the connector code, but excluding eventual 3rd party dependencies
+* `flink-sql-connector-<NAME>` which is an uber JAR ready to use with all the connector 3rd party dependencies.
 
-For more information on how to add dependencies, refer to the build tools sections on [Maven]({{< ref "docs/dev/configuration/maven" >}})
-and [Gradle]({{< ref "docs/dev/configuration/gradle" >}}). 
+The same applies for formats as well. Also note that some connectors, because they don't require 3rd party dependencies,
+may not have a corresponding `flink-sql-connector-<NAME>` artifact.
```

Comment:
```suggestion
The same applies for formats as well. Note that some connectors may not have a corresponding `flink-sql-connector-<NAME>` artifact because they do not require third-party dependencies,.
```